include/internal/quic_txp.h

```diff
@@ -56,7 +56,15 @@ typedef struct ossl_quic_tx_packetiser_args_st {
      *       crypto[QUIC_PN_SPACE_APP] is the 1-RTT crypto stream.
      */
     QUIC_SSTREAM    *crypto[QUIC_PN_SPACE_NUM];
-} OSSL_QUIC_TX_PACKETISER_ARGS;
+
+    /*
+     * Counts of the number of bytes received and sent while in the closing
+     * state.
+     */
+    uint64_t                        closing_bytes_recv;
+    uint64_t                        closing_bytes_xmit;
```

Comment:
These fields seem to be duplicated in the args struct unnecessarily and are never touched by the TXP. They can be removed.

Reply:
Nice catch.  Removed.